packages/client/src/constants.ts

```diff
@@ -16,11 +16,21 @@ export const IMPACT_NFT_PROGRAM_ID = new PublicKey(
   "SUNFT6ErsQvMcDzMcGyndq2P31wYCFs6G6WEcoyGkGc"
 );
 
+export const MARINADE_PROGRAM_ID = new PublicKey(
+  "MarBmsSgKXdrN1egZf5sqe1TMai9K1rChYNDJgjq7aD"
+);
+
 interface BlazeConfig {
   pool: PublicKey;
   bsolMint: PublicKey;
 }
 
+interface MarinadeConfig {
```

Comment:
Is this in order to be able to remove the marinade sdk from our build?

Reply:
That could be a long-term idea. Right now it only reduces our dependency on it. i.e now we're able to derive the bsolTokenAccountAuthority without needing to have called initMarinade() first.